scripts/whats_missing.py

```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env conda-execute
```

Comment:
For context, this was used to figure out what packages we might be missing in `conda-forge` that `defaults` has. Not sure if we have a replacement for this.

Reply:
ok will add back.